querydsl-jpa/src/test/resources/com/querydsl/jpa/testutil/mssql.properties

```diff
@@ -1,5 +1,5 @@
 ## MSSQL
-hibernate.dialect=org.hibernate.dialect.SQLServerDialect
+hibernate.dialect=org.hibernate.dialect.SQLServer2005Dialect

```

Comment:
How come you used `2005` here but `2008` in [Mode#GetSQLTemplates()](https://github.com/querydsl/querydsl/pull/1606/files#diff-38042974cf7ecb54afc1f8a1cf7a718cR22)?